src/main/java/heat/db/DBHelper.java

```diff
@@ -0,0 +1,50 @@
+package heat.db;
+
+import heat.config.DatabaseConfig;
+import heat.utils.ResourceExtractor;
+
+import java.sql.*;
+
+public class DBHelper {
+
+    public static void createUser(User user) {
+        try {
+            final String INSERT_USER = ResourceExtractor.getSql(ResourceExtractor.SQL_INSERT_USER);
+            Connection connection = DatabaseConfig.getDataSource().getConnection();
+            PreparedStatement ps = connection.prepareStatement(INSERT_USER, Statement.RETURN_GENERATED_KEYS);
+
+            ps.setString(1, user.getPassword());
+            ps.setInt(2, user.getGender().index());
+            ps.setDate(3, new Date(user.getBirthDate()));
+            ps.executeUpdate();
+
+            ResultSet rs = ps.getGeneratedKeys();
+            if (rs.next()) {
+                user.setUserId(rs.getInt(1));
+            }
+            rs.close();
```

Comment:
Mb it's better to close `rs` and `ps` in `finally` block. But if you just left it to discuss with other exception cases, then OK))

Reply:
Done.